src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -72,7 +72,15 @@ export default class DiscussionArea {
   addSeekingListener() {
     this.#eventController.addEventListener((timestampMs) => {
       this.seek(timestampMs);
-    }, 'seek');
+    }, 'seek', 'seekAll');
+  }
+
+  /**
+   * Broadcasts a `seekAll` event for a specific `timestampMs` that was clicked.
```

Comment:
This explains what the code does. What does the 'seekAll' broadcast do? I am not sure what this method is supposed to do. So I assume this method is called when the timestamp of a comment is clicked? But that does not explain what the method will do. I would rename the method. Am I understanding correctly?

Reply:
I updated the method name and JSDoc.  Let me know if it's clearer now